### PR TITLE
add services injection

### PR DESCRIPTION
**Pull request by bcroq**
(no service implemented in this pull request)

### CODE REVIEW

**Comment by nagareproject** [comment]
@bcroq Fix the copyright headers please

File: kansha/app/app.py [hunk 1]
```diff
@@ -144,8 +175,17 @@ def go(self, comp):
         user = security.get_user()
         while user is None:
             # not logged ? Call login component
-            comp.call(login.Login(self.app_title, self.app_banner, self.custom_css,
-                                  self.mail_sender, self.cfg, self.assets_manager))
+            comp.call(
+                self._services(
```
Comment (rtxm) [inline kansha/app/app.py]: Where is self._service defined?

Comment (bcroq) [inline kansha/app/app.py]: this is now fixed

**Comment by bcroq** [comment]
@nagareproject copyrights are now fixed

**Comment by rtxm** [comment]
I'm holding this PR for the moment, as we are entering feature freeze for the upcoming new official stable release.

**Pull request merged**